src/Native/MatrixFactorizationNative/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 project (MatrixFactorizationNative)
+add_definitions(-D_SCL_SECURE_NO_WARNINGS)
```

Comment:
Is it part of something else?

Reply:
This is a change that @wschin suggested, in order to make the new libmf library build on my machine.

Reply:
Yes, @yaeldekel's complier is more strict on C++ code about pointer.